docs/releases/4.0.md

```diff
@@ -9,6 +9,12 @@ depth: 1
 
 ## What's new
 
+### Global settings models
+
+The new ``BaseGenericSetting`` base model class allows defining a settings model that applies to all sites rather than just a single site.
```

Comment:
```suggestion
The new `BaseGenericSetting` base model class allows defining a settings model that applies to all sites rather than just a single site.
```